### ISSUE TITLE
論文マラソン第５週目

### ISSUE THREAD

**Issue by rild** [rild/TIL#8]
- 4-2 Fully Character-Level Neural Machine Translation
without Explicit Segmentation

[pdf](https://arxiv.org/pdf/1610.03017.pdf)

---
4-1 で "Lee, 2016" と引用されてたのが目について

**Comment by rild** [comment]
- 4-3 learning rate optimization

[pdf](https://arxiv.org/pdf/1704.04966.pdf)

**Comment by rild** [comment]
- 5-1 Sequence-to-Sequence Neural Net Models for Grapheme-to-Phoneme Conversion

[pdf](https://arxiv.org/abs/1506.00196)

---
DeepVoice: 3.1. Grapheme-to-Phoneme Model

**Comment by rild** [comment]
- 5-2 Empirical Evaluation of Gated Recurrent Neural Networks on Sequence Modeling
Junyoung Chung, Caglar Gulcehre, KyungHyun Cho, Yoshua Bengio

[pdf](https://arxiv.org/abs/1412.3555)

**Comment by rild** [comment]
- 5-3 Persistent RNNs: Stashing Recurrent Weights On-Chip

[link](http://proceedings.mlr.press/v48/diamos16.html)

---
DeepVoice ref

**Comment by rild** [comment]
http://cs231n.github.io/

**Comment by rild** [comment]
http://distill.pub/2017/momentum/

**Comment by rild** [comment]
- 5-4 On the importance of initialization and momentum in deep learning
Sutskever, I., Martens, J., Dahl, G.E. and Hinton, G.E., 2013. ICML (3), Vol 28, pp. 1139—1147.

[PDF](http://proceedings.mlr.press/v28/sutskever13.pdf)

**Comment by rild** [comment]
https://static.googleusercontent.com/media/research.google.com/en//pubs/archive/41539.pdf

スライド 2013

**Comment by rild** [comment]
http://sython.org/demo/icassp2017advtts/demo.html

search with "Japanese Speech Synthesis with DNN "

**Comment by rild** [comment]
- 5-5 [Tacotron: Towards End-to-End Speech Synthesis](https://arxiv.org/pdf/1703.10135.pdf)
[link](https://arxiv.org/abs/1703.10135)

interspeech に投稿された論文探し

**Comment by rild** [comment]
- 5-6 [CHAR2WAV: END-TO-END SPEECH SYNTHESIS](https://mila.umontreal.ca/wp-content/uploads/2017/02/end-end-speech.pdf)

[ICLR](http://www.iclr.cc/doku.php?id=ICLR2017:main&redirect=1)

[他のレポジトリ](https://github.com/arXivTimes/arXivTimes/issues/224)

**Comment by rild** [comment]
- 5-7 Frustratingly Short Attention Spans in Neural Language Modeling

[link](https://arxiv.org/abs/1702.04521)

**Comment by rild** [comment]
- 5-8 
[他のレポジトリ:arXivTimes](https://github.com/arXivTimes/arXivTimes/issues/200)

**Comment by rild** [comment]
論文探し

https://openreview.net/

**Comment by rild** [comment]
全然読めてないまずい

**Comment by rild** [comment]
- 5-9 An Empirical Exploration of Recurrent Network Architectures

[他のレポジトリ:arXivTimes](https://github.com/arXivTimes/arXivTimes/issues/200)

**Comment by rild** [comment]
(Qiita の記事から引っ張って来てる)

http://qiita.com/icoxfog417/items/f170666d81f773e4b1a7

**Comment by rild** [comment]
Interspeech 

ICASSP
